src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx

```diff
@@ -17,70 +17,61 @@ import {
 import { ExpandMore } from '@material-ui/icons';
 import { useTranslation } from 'react-i18next';
 import theme from '../../../../theme';
-
-export interface Contact {
-  name: string;
-  contactId: string;
-  status: string;
-  commitment: string;
-  frequency: string;
-  converted: string;
-  currency: string;
-  donation?: string;
-}
+import { ExpectedDonationRowFragment } from 'pages/accountLists/[accountListId]/reports/GetExpectedMonthlyTotals.generated';
 
 interface Props {
+  accountListId: string;
   title: string;
-  data: Contact[];
+  data: ExpectedDonationRowFragment[];
   donations: boolean;
+  total: number;
+  currency: string;
 }
 
 export const ExpectedMonthlyTotalReportTable: React.FC<Props> = ({
+  accountListId,
   title,
   data,
   donations,
+  total,
+  currency,
 }) => {
   const { t } = useTranslation();
 
-  const totalPartners = () => {
-    return 6;
-  };
-
   const [visible, setVisible] = useState(true);
 
-  const total = () => {
-    return 3920;
-  };
-
   const showTotalPartners = () => {
     if (visible) {
-      return t(' Show ' + totalPartners() + ' Partners');
+      return t(
+        ' Show ' + data.length + ` Partner${data.length > 1 ? 's' : ''}`,
+      );
```

Comment:
```suggestion
      return  data.length > 1 ? t('Show {{partnerCount}} Partners', { partnerCount: data.length }) : t('Show 1 Partner')
```
@OzzieOrca How does this look to you?

Reply:
I was going to suggest i18next's plural feature but if we're extracting text directly from the source code, I'm not sure how you'd specify other keys... This is fine.